services/scan/src/store.ts

```diff
@@ -1045,17 +1060,21 @@ export class Store {
   }
 
   getBallotPageLayoutForMetadata(
-    metadata: BallotPageMetadata
+    metadata: BallotPageMetadata,
+    electionDefinition?: ElectionDefinition
   ): BallotPageLayout | undefined {
-    return this.getBallotPageLayoutsForMetadata(metadata).find(
-      (layout) => layout.metadata.pageNumber === metadata.pageNumber
-    );
+    return this.getBallotPageLayoutsForMetadata(
+      metadata,
+      electionDefinition
+    ).find((layout) => layout.metadata.pageNumber === metadata.pageNumber);
   }
 
   getBallotPageLayoutsForMetadata(
-    metadata: BallotMetadata
+    metadata: BallotMetadata,
+    electionDefinitionFromProps?: ElectionDefinition
```

Comment:
Instead of call this `electionDefinitionFromProps` (since props are more of a React-component-specific term), we could eliminate the need for a different name by specifying the fallback up here:

```ts
getBallotPageLayoutsForMetadata(
  metadata: BallotMetadata,
  electionDefinition: ElectionDefinition = this.getElectionDefinition(),
): BallotPageLayout[] {
```

Reply:
I tried and was stuck in linting purgatory:

![Screen Shot 2022-10-18 at 8 41 14 AM](https://user-images.githubusercontent.com/37960853/196483950-5830f332-cbfd-4b0b-aaee-1588a2fd30db.png)


![Screen Shot 2022-10-18 at 8 41 36 AM](https://user-images.githubusercontent.com/37960853/196483955-651575db-0fc5-4181-95d5-e33302c88af6.png)


![Screen Shot 2022-10-18 at 8 42 15 AM](https://user-images.githubusercontent.com/37960853/196483958-f58e9232-b485-41ea-93d9-02c251e33f63.png)

@arsalansufi I think the ideal solution would be to cache values like this rather than hit the database and parse it every time, long-term.

Reply:
This should work, @adghayes:

```ts
  getBallotPageLayoutsForMetadata(
    metadata: BallotMetadata,
    electionDefinition = this.getElectionDefinition()
  ): BallotPageLayout[] {
```